implementations/app-python/polywrap.yaml

```diff
@@ -0,0 +1,10 @@
+format: 0.3.0
+project:
+  name: app-typescript-wrap-abi-bindgen
+  type: wasm/rust
+source:
+  module: ./Cargo.toml
+  schema: ./polywrap.graphql
+  import_abis:
+    - uri: wrap://ens/wraps.eth:abi-bindgen@0.1
```

Comment:
this can be done in another PR, but we should migrate to using wrapscan.io URIs and deployments.